integration-tests/api/__tests__/totals/orders.js

```diff
@@ -36,137 +37,154 @@ describe("Order Totals", () => {
-      ],
-    })
-
-    const region = await simpleRegionFactory(dbConnection, {
-      gift_cards_taxable: false,
-      tax_rate: 25,
-    })
-
-    const cart = await simpleCartFactory(dbConnection, {
-      id: "test-cart",
-      email: "testnation@medusajs.com",
-      region: region.id,
-      line_items: [],
-    })
-
-    const giftCard = await simpleGiftCardFactory(dbConnection, {
-      region_id: region.id,
-      value: 160000,
-      balance: 160000,
-    })
-
-    const api = useApi()
-
-    await api.post("/store/carts/test-cart/line-items", {
-      quantity: 1,
-      variant_id: "variant_1",
-    })
-    await api.post("/store/carts/test-cart/line-items", {
-      quantity: 1,
-      variant_id: "variant_2",
-    })
-    await api.post("/store/carts/test-cart", {
-      gift_cards: [{ code: giftCard.code }],
-    })
-    await api.post(`/store/carts/${cart.id}/payment-sessions`)
-    const response = await api.post(`/store/carts/test-cart/complete`)
-    expect(response.status).toEqual(200)
-    expect(response.data.type).toEqual("order")
-    const orderId = response.data.data.id
-
-    const { data } = await api.get(`/admin/orders/${orderId}`, {
-      headers: { Authorization: `Bearer test_token` },
-    })
-
-    expect(data.order.gift_card_transactions).toHaveLength(1)
-    expect(data.order.gift_card_transactions).toEqual(
-      expect.arrayContaining([
-        expect.objectContaining({
-          amount: 160000,
-          is_taxable: false,
-          tax_rate: null,
-        }),
-      ])
-    )
-    expect(data.order.gift_card_total).toEqual(160000)
-    expect(data.order.gift_card_tax_total).toEqual(0)
-    expect(data.order.total).toEqual(59000)
+    const db = useDb()
+    await db.teardown()
   })
 
-  test("calculates totals correctly for order with taxable gift card", async () => {
-    await adminSeeder(dbConnection)
-    await simpleProductFactory(dbConnection, {
-      variants: [
-        { id: "variant_1", prices: [{ currency: "usd", amount: 95600 }] },
-        { id: "variant_2", prices: [{ currency: "usd", amount: 79600 }] },
-      ],
-    })
-
-    const region = await simpleRegionFactory(dbConnection, {
-      gift_cards_taxable: true,
-      tax_rate: 25,
-    })
-
-    const cart = await simpleCartFactory(dbConnection, {
-      id: "test-cart",
-      email: "testnation@medusajs.com",
-      region: region.id,
-      line_items: [],
-    })
-
-    const giftCard = await simpleGiftCardFactory(dbConnection, {
-      region_id: region.id,
-      value: 160000,
-      balance: 160000,
-    })
-
-    const api = useApi()
-
-    await api.post("/store/carts/test-cart/line-items", {
-      quantity: 1,
-      variant_id: "variant_1",
+  describe("GET /admin/orders/:id/totals", () => {
+    beforeEach(async () => {
+      await adminSeeder(dbConnection)
+
+      await simpleProductFactory(dbConnection, {
+        variants: [
+          { id: "variant_1", prices: [{ currency: "usd", amount: 95600 }] },
```

Comment:
suggestion: I would say that you can declare a const at the beginning of the describe and store the variant ids using IdMap. Then you can use that const everywhere and it would be less error pron if we need to change the id for any reason